app/models/session.rb

```diff
@@ -34,6 +34,23 @@ def expire(ttl)
     super(ttl)
   end
 
+  def user
+    User.find(uuid)
+  end
+
+  def cookie_data
+    return nil if user.blank?
+    {
+      'patientIcn' => user.icn,
+      'mhvCorrelationId' => user.mhv_correlation_id,
+      'expirationTime' => ttl_in_time.iso8601(0)
+    }
+  end
+
+  def ttl_in_time
```

Comment:
Did you forget to have `ttl` as an arg here?

Reply:
Never mind, I see it's defined in `RedisStore`.

Reply:
no, ttl is an attribute on RedisStore already. I'm converting TTL to actual time... Otherwise it will be N < 1800. I want CurrentTime + N